radio/src/targets/nv14/battery_driver.cpp

```diff
@@ -175,7 +175,7 @@ void handle_battery_charge(uint32_t last_press_time)
     return;
   }
 
-  if(updateTime == 0 || ((get_tmr10ms() - updateTime) >= 50))
+  if(updateTime == 0 || ((timersGetMsTick() - updateTime) >= 500))
```

Comment:
Can we change the 500 to a constant, rather than a magic number for half second? ;) For both NV14/PL18, etc... can be in the common PR if not already there...